src/renderer/Inventory/Item/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import ItemRenderer from '../../Item';
+import { ItemRendererProps } from '../../Item/types';
+
+import './item.css';
+
+interface InventoryItemProps extends ItemRendererProps {
+  onClick(): void;
```

Comment:
It could be better as:

```suggestion
  onClick(item: Item): void;
```

as then the implementor could theoretically cache a single function for multiple of these.